doc/nrf/releases/release-notes-latest.rst

```diff
@@ -66,21 +66,50 @@ sdk-zephyr
 
 .. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
 
-The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``7a3b253ced``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repository up to and including ``35264cc214fd``, plus some |NCS| specific additions.
 
 For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
 
 .. code-block:: none
 
-   git log --oneline 7a3b253ced ^v2.3.0-rc1-ncs1
+   git log --oneline 35264cc214fd ^v2.4.0-ncs1
 
 For a complete list of |NCS| specific commits, run:
 
 .. code-block:: none
 
-   git log --oneline manifest-rev ^7a3b253ced
-
-The current |NCS| release is based on Zephyr 2.4.0.
-See the :ref:`Zephyr 2.4.0 release notes <zephyr:zephyr_2.4>` for a list of changes.
-
-For the list of the most recent additions specific to |NCS|, see :ref:`ncs_release_notes_140`.
+   git log --oneline manifest-rev ^35264cc214fd
+
+The current |NCS| release is based on the development series for Zephyr v2.5.0.
```

Comment:
```suggestion
The current |NCS| release is based on Zephyr v2.4.99.
```
Let's keep it simple - mentioning Zephyr 2.5.0 is just gonna make things more confusing.